controller/api/destination/external-workload/queue_metrics.go

```diff
@@ -23,6 +23,7 @@ const (
 	UnfinishedWorkKey          = "unfinished_work_seconds"
 	LongestRunningProcessorKey = "longest_running_processor_seconds"
 	RetriesKey                 = "retries_total"
+	DropsTotal                 = "drops_total"
```

Comment:
a very inconsequential suggestion but wdyt of staying consistent with the rest of the naming here?

```suggestion
	DropsTotalKey                 = "drops_total"
```